chapters/0-introduction.tex

```diff
@@ -201,43 +207,45 @@ \subsection{Code examples}
 but you should reference Stata help-files by writing \texttt{help [command]}
 whenever you do not understand the command that is being used.
 We hope that these snippets will provide a foundation for your code style.
-Providing some standardization to Stata code style is also a goal of this team;
-we provide our guidance on this in the Stata Style Guide in the Appendix.
+Providing some standardization to Stata code style is also a goal of this team.
 
 \section{Outline of this book}
 
 This book covers each stage of an empirical research project, from design to publication.
 We start with ethical principles to guide empirical research,
-focusing on research transparency and the right to privacy.
-In Chapter 1, we outline a set of practices that help to ensure
-research participants are appropriately protected and
-research consumers can be confident in the conclusions reached.
-Chapter 2 will teach you to structure your data work to be efficient,
-collaborative and reproducible.
-It discusses the importance of planning data work at the outset of the research project --
+focusing on research reproducibility, transparency, and credibility.
+In Chapter 1, we outline a set of practices that help to ensure that
+research consumers can be confident in the conclusions reached,
+and research work can be assumed and verified to be reliable.
+Chapter 2 will teach you to structure your data work to be organized
+and collaborative, while ensuring the privacy and security of research participants.
+It discusses the importance of planning these data tools and structures work at the outset of the research project --
 long before any data is acquired -- and provides suggestions for collaborative workflows and tools.
```

Comment:
```suggestion
long before any data is acquired --;
and provides suggestions for collaborative workflows and tools.
```

Reply:
This doesn't seem right, is this a typo @luizaandrade?

Reply:
yes, it was just meant to be a new line :)